### ISSUE TITLE
Sectors copied back and forth between nodes at PC1

### ISSUE THREAD

**Issue by hengic** [filecoin-project/lotus#3399]
**Describe the bug**
I have two nodes only doing PC1. Each of them can process 14 tasks maximum in parallel. If I pledged more than 28 sectors, the rest sectors will be fetched between nodes back and forth.

**To Reproduce**
1. prepare two nodes only doing PC1
2. pledge more sectors than they can afford.
3. the rest sectors are fetched between nodes. 

**Expected behavior**
The sector stays where it was until any node has the ability to process it.

**Screenshots**
<img width="500" alt="Screen Shot 2020-08-30 at 10 11 52 AM" src="https://user-images.githubusercontent.com/22890997/91649718-410d8880-eaa9-11ea-9ec4-a032003ca0e1.png">
see sector 56,57,60,62, it keeps getting between the node of fc-p1-1 and fc-p1-2

**Version (run `lotus version`):**
Daemon:  0.5.6+git.606a58bc.dirty+api0.13.0
Local: lotus version 0.5.6+git.606a58bc.dirty

**Additional context**

**Comment by rjan90** [comment]
I think this issue has been fixed in newer versions of Lotus, and can be closed now? #rengjøring

**Comment by jennijuju** [comment]
Please open a new ticket if you are running into this issue on the latest version of lotus!
